chain/types/tipset.go

```diff
@@ -196,8 +196,19 @@ func (ts *TipSet) MinTicket() *Ticket {
 }
 
 func (ts *TipSet) MinTimestamp() uint64 {
-	minTs := ts.Blocks()[0].Timestamp
-	for _, bh := range ts.Blocks()[1:] {
+	if ts == nil {
+		return 0
+	}
+
+	blks := ts.Blocks()
+
+	if len(blks) == 0 {
```

Comment:
How can you ever have an instance of TIpSet with no blocks? I wouldn't expect that to be possible

Reply:
@vyzo -- could you explain this?

Reply:
conf tests bomb wihout this iirc.

Reply:
also, null rounds?

Reply:
Null rounds shouldn't ever be represented as tipsets with no blocks; Null-round generally means that the tipset at that epoch doesn't exist - and the next tipset that does exist links straight to first epoch with blocks